test/reducers/minions.js

```diff
@@ -14,13 +14,42 @@ describe('minions reducer', () => {
       minionsById: {},
     });
   });
+  it('should handle FREEZE', () => {
+    const action1 = freeze(1, 'minionId1');
+    const action2 = freeze(2, 'minionId2');
+    const action3 = freeze(3, 'minionId1', 'minionId2', 'heroId1', 'heroId2');
+    let minionState = minions(initialMinionState, action1);
+    expect(minionState.minionsById.minionId1.frozenFor).to.equal(1);
+    minionState = minions(minionState, action2);
+    expect(minionState.minionsById.minionId2.frozenFor).to.equal(2);
+    minionState = minions(minionState, action3);
+    expect(minionState.minionsById.minionId1.frozenFor).to.equal(3);
+    expect(minionState.minionsById.minionId2.frozenFor).to.equal(3);
+  });
   it('should handle KILL', () => {
     const action1 = {
       type: KILL,
       characterId: 'minionId1',
     };
     const minionState = minions(initialMinionState, action1);
-    expect(minionState.minionsById).to.eql({});
+    expect(minionState.minionsById).to.eql({
```

Comment:
Change this to test the number of keys `minionState.minionsById` has rather than testing for equality